asm/istio/expansion/README.md

```diff
@@ -0,0 +1,19 @@
+# Anthos Service Mesh Expansion
+
+There are currently a few files in this directory that supports expansion to the
+mesh.
+
+- `gen-eastwest-gateway.sh` can be used to generate a custom `IstioOperator`
+configuration for the expansion gateway deployment. Please refer to the current
+Anthos Service Mesh [user guide](https://cloud.google.com/service-mesh/docs/gke-on-prem-install-multicluster-vmware)
+on how to use the script.
+
+- `vm-eastwest-gateway.yaml` is a generated `IstioOperator` configuration for
+the expansion gateway deployment to be used when adding VM workloads on the same
+network to the mesh.
+
+- `expose-istiod.yaml` is used to expose the control plane for discovery from
```

Comment:
Maybe note that the `expose-*.yaml` files have to be used in combo with the `vm-eastwest-gateway.yaml` and handle `istiod` and not `istiod-asm-${rev}`

Reply:
It actually handles revisioned istiod using kpt. I prefer not to document too much of that usage until we consolidate more on expansion gateway usage.